### PR TITLE
Update metadata attributes

### LINKED ISSUE #42: Substitute PID with IRI for attributes
We are now following nomenclature from SEEK
So in
```json
"pid": null,
"name": "Input",
"description": "Existing Samples in DataHub that represent input for this protocol.",
"dataType": "Registered Sample (multiple)",
"required": true,
"isaTag": null
``` 

`pid` must become `iri`

### LINKED ISSUE #43: For template metadata substitute template_version with version
The [schema](https://github.com/seek4science/seek/blob/master/lib/seek/isa_templates/template_schema.json) looks for `version` and not `template_version`

### PR DESCRIPTION
**Pull request by kdp-cloud**
The templates on the main branch were deprecated and could not be uploaded in SEEK anymore. In order to comply to the latest SEEK configurations, some JSON attributes had to be updated.

### CODE REVIEW

**Comment by kdp-cloud** [comment]
Additions:
- I will add to change from 'study' to 'study source'.
- I will add to change from 'assay' to 'study assay'.
- I will add a (already updated) minimalistic test template.

**Comment by floradanna** [comment]
> Additions:
> 
> * I will add to change from 'study' to 'study source'.

Correct

> * I will add to change from 'assay' to 'study assay'.

This doesn't seem correct. Do you mean changing from 'study' to 'study sample' ?

> * I will add a (already updated) minimalistic test template.

**Comment by kdp-cloud** [comment]
I changed 'study assay' to 'study sample' in the [latest commit](https://github.com/ELIXIR-Belgium/ISA-merger/pull/44/commits/ce4809123f4f1e4089964cd43c5227322f3ed874).
I'll make the necessary changes in the application.

**Review by floradanna — CHANGES_REQUESTED** [review]
I am not sure that it is correct. Not all "assay" should be "study sample". It depends on the protocol type.

- if there is no protocol, and there is isa_tag=source, then level = study source
- if protocol type = sample collection, then level = study sample
- for any other protocol type, level = assay

**Review by floradanna — APPROVED** [review]

File: templates/isa test/minimalistic_isa_test_v1.0.1.json [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+					"CVList": null,
+					"isaTag": "source_characteristic"
+				}
+			]
+		},
+		{
+			"metadata": {
+				"name": "ISA Minimalistic Test 2",
+				"group": "ISA Minimalistic Test",
+				"group_order": 1,
```
Comment (rabuono) [inline templates/isa test/minimalistic_isa_test_v1.0.1.json]:
```suggestion
				"group_order": 2,
```

File: templates/isa test/minimalistic_isa_test_v1.0.1.json [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+				"temporary_name": "1_ISA_Test_Source",
```
Comment (rabuono) [inline templates/isa test/minimalistic_isa_test_v1.0.1.json]:
```suggestion
				"temporary_name": "2_ISA_Test_Sample",
```

File: templates/isa test/minimalistic_isa_test_v1.0.1.json [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+				"isa_technology_type": null,
+				"isa_protocol_type": null,
+				"repo_schema_id": null,
+				"organism": "any",
+				"level": "study sample"
+			},
+			"data": [
+				{
+					"iri": null,
+					"name": "Source Name",
```
Comment (rabuono) [inline templates/isa test/minimalistic_isa_test_v1.0.1.json]:
```suggestion
					"name": "Sample Name",
```

File: templates/isa test/minimalistic_isa_test_v1.0.1.json [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+					"description": "Sources are considered as the starting biological material used in a study.",
+					"dataType": "String",
+					"title": true,
+					"required": true,
+					"isaTag": "source"
```
Comment (rabuono) [inline templates/isa test/minimalistic_isa_test_v1.0.1.json]:
```suggestion
					"isaTag": "sample"
```

File: templates/isa test/minimalistic_isa_test_v1.0.1.json [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+				},
+				{
+					"iri": null,
+					"name": "Source Characteristic 1",
+					"description": "A characteristic of the source",
+					"dataType": "String",
+					"required": true,
+					"ontology": null,
+					"CVList": null,
+					"isaTag": "source_characteristic"
```
Comment (rabuono) [inline templates/isa test/minimalistic_isa_test_v1.0.1.json]:
```suggestion
					"isaTag": "sample_characteristic"
```

File: templates/isa test/minimalistic_isa_test_v1.0.1.json [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+					"description": "Sources are considered as the starting biological material used in a study.",
+					"dataType": "String",
+					"title": true,
+					"required": true,
+					"isaTag": "source"
+				},
+				{
+					"iri": null,
+					"name": "Source Characteristic 1",
+					"description": "A characteristic of the source",
```
Comment (rabuono) [inline templates/isa test/minimalistic_isa_test_v1.0.1.json]:
```suggestion
					"description": "A characteristic of the sample",
```

File: templates/isa test/minimalistic_isa_test_v1.0.1.json [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+					"name": "Source Name",
+					"description": "Sources are considered as the starting biological material used in a study.",
+					"dataType": "String",
+					"title": true,
+					"required": true,
+					"isaTag": "source"
+				},
+				{
+					"iri": null,
+					"name": "Source Characteristic 1",
```
Comment (rabuono) [inline templates/isa test/minimalistic_isa_test_v1.0.1.json]:
```suggestion
					"name": "Sample Characteristic 1",
```

File: templates/isa test/minimalistic_isa_test_v1.0.1.json [hunk 8]
```diff
@@ -0,0 +1,143 @@
...
+				"isa_config": null,
+				"isa_measurement_type": null,
+				"isa_technology_type": null,
+				"isa_protocol_type": null,
+				"repo_schema_id": null,
+				"organism": "any",
+				"level": "study sample"
+			},
+			"data": [
+				{
```
Comment (rabuono) [inline templates/isa test/minimalistic_isa_test_v1.0.1.json:56]: Best to add a `input` and a `protocol` here before the Sample below

**Review by rabuono — CHANGES_REQUESTED** [review]
One of the ISA minimalistic templates was tagged as `study sample` but was describing a source. I made some changes but it is missing  an input and a protocol of sample collection.

File: templates/isa test/minimalistic_isa_test_v1.0.1.json (hunk 8 above)
Comment (kdp-cloud) [reply templates/isa test/minimalistic_isa_test_v1.0.1.json:56]: Done in commit [a04bb31](https://github.com/ELIXIR-Belgium/ISA-merger/pull/44/commits/a04bb31f4880d394e24eb2d256c321e3562cede2).

**Pull request merged**